### ISSUE TITLE
Authorization and credentials roles

### ISSUE THREAD

**Issue by lcdunne** [ocpi/ocpi#509]
Suppose there are 3 platforms, A, B, and C. Platforms A and B are integrated via the handshake, as are platforms A and C (but not B and C). Assuming platforms B and C have no business agreement, what prevents platform C from simply including a credentials role object containing platform B's country code & party id, and providing these credentials to platform A? It seems to me that this would potentially allow platform C unauthorised access to platform B's data.

Maybe I am overlooking something obvious with this, but my only guess is that there is that it is left down to the specific auth implementation on platform A to prevent this, but that no such recommendation is made within OCPI.

**Comment by reinierl** [comment]
You are correct. There is no mechanism in OCPI to authenticate the parties on a platform. OCPI only describes authentication of platforms (with the Authentication Token).

Authorization of platforms and both authentication and authorization of parties are thereby left up to platforms to  design and implement by themselves.
